services/movie_session.py

```diff
@@ -42,3 +42,18 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `get` method can raise a `DoesNotExist` exception if no movie session is found with the given ID. Consider handling this exception to prevent potential crashes.